src/ViewRenderer.php

```diff
@@ -116,7 +117,7 @@ public function withLayout(string $layout): self
     public function withAddedInjections(array $injections): self
     {
         $new = clone $this;
-        $new->injections = array_merge($this->injections, $injections);
+        $new->injections = \array_merge($this->injections, $injections);
```

Comment:
use function in header instead of this?

Reply:
By the way, why do you specify a namespace for these functions at all?
If for the purpose of improving performance, then this will not work for many functions from this PR.
For overload protection?

Reply:
@roxblnfk first of all for clean code, imo usage backslash don't look a beautiful

Reply:
I'm not talking about moving the namespaces to `use`, but about the fact that the `\` namespace  is generally superfluous here

Reply:
@roxblnfk i guess you are right. I didn't even think about it

Reply:
> I'm not talking about moving the namespaces to `use`, but about the fact that the `\` namespace is generally superfluous here

Not superfluous. Firstly, it uniquely defines the search area (it is not required to search recursively), and secondly, it protects against redistribution.